Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -512,6 +512,10 @@ private void OnKeyDown(object sender, KeyEventArgs e)
                         e.Handled = true;
                     }
                     break;
+                case Key.F1:
+                    _viewModel.OpenQuickLook.Execute(null);
+                    e.Handled = true;
+                    break;
```

Comment:
I think we have registered another `F1` keystork so you may want to replace that with OpenQuickLook instead of setting that here.

Reply:
Done